CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.0.2 (TBD)
```

Comment:
Is there any blocker for this release? Or should we change `TBD` to today?

Reply:
No blocker, waiting for a approval then I'll update the date